app/src/main/res/layout-land/administrator_controls_app_information_view.xml

```diff
@@ -18,14 +18,13 @@
       android:layout_width="match_parent"
       android:layout_height="wrap_content"
       android:background="@color/mid_grey_30"
-      android:fontFamily="sans-serif-medium"
+      style="@style/Subtitle2"
```

Comment:
That should be `style="@style/Caption` because the TextView had:  
```
android:fontFamily="sans-serif-medium"
android:textSize="14sp"
```
And these attributes can be found in `Caption`:  
```
<style name="Caption" parent="TextView.Common1">
    <item name="android:fontFamily">sans-serif-medium</item>
    <item name="android:textSize">14sp</item>
    <item name="fontFamily">sans-serif-medium</item>
  </style>
```
not in `Subtitle2`:
```
<style name="Subtitle2" parent="TextView.Common1">
    <item name="android:fontFamily">sans-serif</item>
    <item name="android:textSize">14sp</item>
    <item name="fontFamily">sans-serif</item>
  </style>
```

Reply:
resolving this conversation because the code now is correct